compiler/x/i386/codegen/TreeEvaluatorTable.hpp

```diff
@@ -39,13 +39,27 @@
    TR::TreeEvaluator::bloadEvaluator,                   // TR::bload
    TR::TreeEvaluator::sloadEvaluator,                   // TR::sload
    TR::TreeEvaluator::lloadEvaluator,                  // TR::lload
-   TR::TreeEvaluator::iloadEvaluator,                   // TR::iloadi
-   TR::TreeEvaluator::floadEvaluator,                   // TR::floadi
-   TR::TreeEvaluator::dloadEvaluator,                   // TR::dloadi
+   TR::TreeEvaluator::irdbarEvaluator,                 // TR::irdbar
+   TR::TreeEvaluator::frdbarEvaluator,                 // TR::frdbar
+   TR::TreeEvaluator::drdbarEvaluator,                 // TR::drdbar
+   TR::TreeEvaluator::ardbarEvaluator,                 // TR::ardbar
```

Comment:
It's just the design for static rdbar changed a little bit (because of the extra child) so I wanted to wait until we agree on the x evaluators first and apply them to other platforms.